elasticsearch/run.sh

```diff
@@ -28,4 +39,24 @@ else
 	exit 1
 fi
 
-exec /usr/share/elasticsearch/bin/elasticsearch
+ln -s /usr/share/elasticsearch/config/${CLUSTER_NAME}.truststore /usr/share/elasticsearch/config/${CLUSTER_NAME}.truststore.jks
+cp /etc/elasticsearch/secret/admin-jks /usr/share/elasticsearch/config/admin.jks
+
+TIMES=60
+function waitForES() {
+  for (( i=1; i<=$TIMES; i++ )); do
+    # test for ES to be up first
+		# don't provide a client cert since ES doesn't validate us correctly
```

Comment:
@ewolinetz checking if I understand correctly: Are you saying that when we try to push index templates to ES in `run.sh` we can run into issues because SG did initialize this node correctly?